static/app/components/events/interfaces/threads/threadSelector/option.tsx

```diff
@@ -23,11 +23,30 @@ type ThreadInfo = {
 };
 
 const Option = ({id, details, name, crashed, crashedInfo}: Props) => {
-  const {label = `<${t('unknown')}>`, filename = `<${t('unknown')}>`} = details;
+  const {label = `<${t('unknown')}>`} = details;
```

Comment:
@priscilawebdev Is there a cleaning way of declaring this now that I deleted `filename` ?

Reply:
yes one option would be 

`const label = details.label ?? "<${t('unknown')}>";`

please replace the quotes with "`"...I just used doubled quotes here because otherwise the suggestion wouldn't be correct